drawOscilloscope.js

```diff
@@ -1,9 +1,9 @@
 function drawOscilloscope(samples, x, y, rectWidth, rectHeight) {
-  var distanceFromTop = 400;
-  var distanceFromLeft = 20;
+  var distanceFromTop = height / 2;
+  var distanceFromLeft = width / 72;
```

Comment:
I'm unsure about this naming scheme now. It's not really consistent with our other namings. Maybe `xTranslate` and `yTranslate`?